src/WordSelector/WordSelector.test.js

```diff
@@ -8,6 +8,6 @@ describe('WordSelector', () => {
 
     render(<WordSelector />);
 
-    expect(screen.getByText('Your opponent is thinking of a word...')).toBeInTheDocument();
+    expect(screen.getByText('Type a word for your opponent to guess!')).toBeInTheDocument();
```

Comment:
Both of these will get conditionally rendered based upon which user isGuesser. Still haven't gotten to the functionality of that yet, but it should be the next day or so.